brain/talk.js

```diff
@@ -0,0 +1,61 @@
+const Sentiment = require('sentiment');
+const asciilib = require('asciilib');
+const _ = require('lodash');
+const {
+  sentimentalThought,
+} = require('./thoughts');
+
+const sentiment = new Sentiment();
+
+const bannedWords = [
+  'a',
+  'i',
+  'it',
+  'all',
+  'for',
+  'the',
+  'me',
+  'of',
+  'in',
+  'im',
+  'on',
+  'up',
+];
+
+const respondEmotionally = (action) => {
```

Comment:
this function is too long and does too many different things

Reply:
need to break apart the sentiment analysis. we are analyzing text too soon since we can match an asciilib keyword before getting to `respondEmotionally`

Reply:
This is definitely a Real Big Function, and it's taking me a while to parse through. I agree that it would be good if it could be broken up a bit

Reply:
yeah pls break it up

Reply:
thats the best i can do for now -_-